src/images/badges/made-with-out-pants.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please set the font color to white, and the background color to `#ef4041` to work more consistently with other badges.

![image](https://user-images.githubusercontent.com/8878152/92506856-166fae00-f1d4-11ea-940d-905cf1ff8d21.png)